recipe/meta.yaml

```diff
@@ -17,85 +17,90 @@ build:
 
 requirements:
   build:
+    # Upstream pandas says that 2022 because they use C11.
```

Comment:
I would move this into the conda_build_config file.

Reply:
Ok will do

Reply:
Done. Please provide an approval when you can @Jrice1317. Thanks!